### DISCUSSION TITLE
Summation of all records in the table

### DISCUSSION THREAD

**Discussion by alukardua1** [rappasoft/laravel-livewire-tables#1025] [Feature Requests]
I output the sum of a certain field to secondaryHeader, but there is a nuance $rows->sum(field) sums up only on the current page.
How can I make a summation across all pages.

**Comment by lrljoe** [comment]
As a user, I'd expect the totals in a given page to total the displayed items only.

If you're looking to total/sum across your entire dataset regardless of pagination, then I'd recommend that you select the sum as an additional field, and you should get your desired result.

**Comment by alukardua1** [comment]
I did this, but I still need the amount to change, depending on the filters applied.
That is, all rows have a sum of 1033, and after sampling by the filter, for example, 655 should decrease.

**Comment by lrljoe** [comment]
I understand the use case, although I'd disagree with the approach from a user perspective, as a user, showing me a sum of things I can't see would frankly confuse me, and I'd probably assume that the product can't add up.

However, you'd just need to do a withSum() in your Builder query, constrained with your query and use that value in the relevant place, (i.e. wherever you wish), I'm just not completely sure that I'm seeing what the new feature or problem is here?
